src/main/resources/api/data-repository-openapi.yaml

```diff
@@ -7998,28 +8004,6 @@ components:
       schema:
         type: string
         format: uuid
-    SnapshotPolicyName:
```

Comment:
Using a shared type like this seems like a better approach, is there a reason why it won't work here?